instance/models/mixins/utilities.py

```diff
@@ -26,22 +26,22 @@
 from django.views.debug import ExceptionReporter
 
 
-class EmailInstanceMixin(object):
+class EmailMixin:
     """
-    An instance class that can send emails
+    An AppServer class that can send emails
     """
     class EmailSubject(object):
         """
         Class holding email subject constants
         """
-        PROVISION_FAILED = u"Instance {instance_name} ({instance_url}) failed to provision"
+        PROVISION_FAILED = "AppServer {name} ({instance_name}) failed to provision"
 
     class EmailBody(object):
         """
         Class holding email body constants
         """
-        PROVISION_FAILED = u"Instance {instance_name} failed to provision.\n" \
-                           u"Reason: {reason}\n"
+        PROVISION_FAILED = "Instance {instance_name} AppServer {name} failed to provision.\n" \
```

Comment:
@bradenmacdonald `AppServer {name} for Instance {instance_name} failed to provision` would be even friendlier :)